bin/ci

```diff
@@ -3,9 +3,10 @@
 
 require_relative "../loader"
 require "time"
+require "optparse"
 
-def main
-  hetzner_server_st = Prog::Test::HetznerServer.assemble
+def main(options)
+  hetzner_server_st = Prog::Test::HetznerServer.assemble(vm_host_id: options[:vm_host_id])
```

Comment:
should ubid input be supported also, rather than obliging the growing of a split practice?

Reply:
I added this feature. It converts UBID to UUID while parsing arguments.